pkg/sm/plugins/ufm/ufm_test.go

```diff
@@ -27,18 +31,23 @@ var _ = Describe("Ufm Subnet Manager Client plugin", func() {
 	})
 	Context("newUfmPlugin", func() {
 		It("newUfmPlugin ufm plugin", func() {
-			conf := &config.SubnetManagerPluginConfig{Ufm: config.UFMConfig{
-				Username: "admin", Password: "123456", Address: "127.0.0.1", HttpSchema: "http"}}
-			plugin, err := newUfmPlugin(conf)
+			Expect(os.Setenv("UFM_USERNAME", "admin")).ToNot(HaveOccurred())
+			Expect(os.Setenv("UFM_PASSWORD", "123456")).ToNot(HaveOccurred())
+			Expect(os.Setenv("UFM_ADDRESS", "1.1.1.1")).ToNot(HaveOccurred())
+			Expect(os.Setenv("UFM_HTTP_SCHEMA", "http")).ToNot(HaveOccurred())
+			defer os.Clearenv()
```

Comment:
the os.Clearenv should be in an AfterEach clause so that when you write a test you dont need to worry about it